pkg/client/interfaces.go

```diff
@@ -60,13 +61,16 @@ type E2EConfig struct {
 // RunConfig are the input options for running Sonobuoy.
 type RunConfig struct {
 	GenConfig
+	Wait        bool
+	WaitTimeout time.Duration
 }
 
 // DeleteConfig are the input options for cleaning up a Sonobuoy run.
 type DeleteConfig struct {
 	Namespace  string
 	EnableRBAC bool
 	DeleteAll  bool
+	Wait       bool
```

Comment:
WaitTimeout should have been added here; in which case my previous comment about it not being customizable is false. That gets plumbed down; just the polling interval is not.